<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-47P2MTB" failures="1" tests="14" name="test.SwagLabs1LoginPageTest" time="73.116" errors="0" timestamp="2023-09-28T13:13:49 IST" skipped="0">
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="4.379"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="6.085">
    <failure message="expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/inventory.html] but found [https://www.saucedemo.com/]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.SwagLabs1LoginPageTest.doLogin_1ValidId2ValidPwd_OpenedHomepage(SwagLabs1LoginPageTest.java:57)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- doLogin_1ValidId2ValidPwd_OpenedHomepage -->
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="8.372"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1ValidId2ValidPwd_OpenedHomepage" time="4.601"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1ValidId2InvalidPwd_FailedLoginAndVerifyErrorMSG" time="4.898"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1InvalidId2validPwd_FailedLoginAndVerifyErrorMSG" time="4.737"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1InvalidId2InvalidPwd_FailedLoginAndVerifyErrorMSG" time="4.323"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1EmptyId2GivenPwd_FailedLoginAndVerifyWarningMSG" time="4.885"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1EmptyPwd2GivenId_FailedLoginAndVerifyWarningMSG" time="3.507"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_1EmptyId2EmptyPwd_FailedLoginAndVerifyWarningMSG" time="2.899"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_Clickable_CompletedLogin" time="3.447"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_SwagLabsLogo_DisplayedLogoAndVerifyText" time="2.718"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_ErrorBoxCancelButton_ClosedErrorOrWarningMSG" time="14.630"/>
  <testcase classname="test.SwagLabs1LoginPageTest" name="doLogin_PlaceHolderText_Displayed" time="3.635"/>
</testsuite> <!-- test.SwagLabs1LoginPageTest -->
